app/models/financial_aid/my_financial_aid_summary.rb

```diff
@@ -1,5 +1,6 @@
 module FinancialAid
   class MyFinancialAidSummary < UserSpecificModel
+    include ActionView::Helpers::NumberHelper
```

Comment:
@marcpechaitis - Do we need [this include](https://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html)? It doesn't appear that you've added any method calls from those helpers.

Reply:
@redconfetti yep that's a good catch.  I accidentally left that in when I was playing around with the formatting of the numbers used as dollars.  I have removed